editor/src/components/menubar/menubar.tsx

```diff
@@ -171,6 +171,17 @@ export const Menubar = betterReactMemo('Menubar', () => {
   const previewURL =
     projectId == null ? '' : shareURLForProject(FLOATING_PREVIEW_BASE_URL, projectId, projectName)
 
+  const entireStateRef = useRefEditorState((store) => store)
+
+  const jsxMetadata = useRefEditorState((store) => {
+    return store.editor.jsxMetadata
+  })
+
+  const printEditorState = React.useCallback(() => {
```

Comment:
I added a new PPP button to the Left Menu that prints the editor state and the metadata. This was necessary because the previous (very useful!) print button was only visible with everything deselected. But deselection now clears the focus path

Reply:
But why "PPP"?

Reply:
gotta keep it cryptic!